include/superior_mysqlpp/types/datetime.hpp

```diff
@@ -14,14 +14,39 @@
 
 namespace SuperiorMySqlpp
 {
+    /**
+     * Class Datetime offers automatic initialization and updating to the current date and time. 
+     * The supported range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59'.
```

Comment:
You didn't mention second fractions here.

Reply:
ACK